src/components/Select/Select.tsx

```diff
@@ -0,0 +1,66 @@
+import { Menu, MenuButton, MenuOptionGroup, MenuItemOption, MenuList } from '@chakra-ui/menu';
+import { Button } from '@chakra-ui/button';
+import { FormControl, FormLabel } from '@chakra-ui/form-control';
+import { ReactElement, useState } from 'react';
+import { ChevronDownIcon } from '@chakra-ui/icons';
+
+export interface SelectOption {
+  id: string;
+  label: string;
+  help?: string;
+}
+
+export interface ISelectProps {
+  formLabel?: string;
+  options: SelectOption[];
+  defaultOption: string;
+  onOptionSelected: (id: string) => void;
+}
+
+export const Select = ({ formLabel, options, defaultOption, onOptionSelected }: ISelectProps): ReactElement => {
```

Comment:
Why use this custom component instead of Chakra's built-in `Select`? https://chakra-ui.com/docs/form/select

Reply:
Chakra's select dropdown is native. Here I am using Chakra's menu as dropdown with a customized label.

Reply:
Looks like in the next PR, I updated this component to use ReactSelect rather than Chakra's menu.